src/transport/grpc/mod.rs

```diff
@@ -411,7 +442,10 @@ mod test {
 
     use super::super::{test::mock_shared_device, Publish, Receive, ReceivedEvent};
 
-    use super::{Grpc, GrpcReceivePayload, GrpcTransportError, NodeData};
+    use super::{
+        Grpc, GrpcReceivePayload, GrpcTransportError, MessageHubClientWithInterceptor, NodeData,
+        NodeIdInterceptor,
+    };
```

Comment:
Also for the use super before this can be collapsed in

```suggestion
    use super::*;
```